magento/magento2
# Renamed "Add Block Names to Hints" config setting to represent what it actually does

This config name is notoriously misleading as it does not show the block name at all, instead it shows the block class type. Worse still there is a question about this in the Magento2 certification exam which is utterly confusing due to the poor naming of the config setting.

### Description
I have updated the label of this config setting to state what it actually does when enabled, displaying the block class type on the hints.

### Fixed Issues (if relevant)
n/a

### Manual testing scenarios
- Login to admin, view updated configuration label.
- Set config to "Yes", confirm block class types are still being shown on template hints.

### Contribution checklist
